projects/step-core/src/lib/modules/table/components/table/table.component.ts

```diff
@@ -242,4 +255,12 @@ export class TableComponent<T> implements AfterViewInit, OnChanges, OnDestroy, T
       this.setupDatasource(cDatasource.currentValue);
     }
   }
+
+  exportAsCSV(): void {
+    if (!this.tableDataSource) {
+      console.log('No datasource for export');
```

Comment:
In general I would say hiding errors in the console is not best practise, but I think this should normally never happen so it is fine. But please use `console.error` for these cases.